README.md

```diff
@@ -0,0 +1,137 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Tests](#tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Budget App] <a name="about-project"></a>
+
+**[Budget App]** is a project created to manage and display entries in a library using Object Oriented Programming with Ruby.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>The 'Budget App' app was built using Ruby on Rails, and PostgreSQL</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+    <li>PostgreSQL</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Add different types of entries
+- Organize them into a collection
+- Preseve and load data
+- List and display the different categories
+- PostgreSQL
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## Live Demo <a name="live-demo"></a>
+
+Visit the site live [here.](https://mysite-28qd.onrender.com)
+
+You can see the video presentation for this project [here.](https://www.loom.com/share/8f12e2777cdc4a198036debc24bcd1)
+
```

Comment:
- [ ] The links you have provided are not functioning correctly as shown in the images below please kindly work on this :+1: 
![vid](https://github.com/banzaman/budget_app/assets/89055502/ee86e27a-4f96-45ad-9ea1-6d39b8ebb5b1)
![vi](https://github.com/banzaman/budget_app/assets/89055502/a8576b3c-827e-479c-9608-0efcd68b825b)